scripts/auth_init.ps1

```diff
@@ -0,0 +1,9 @@
+. ./scripts/loadenv.ps1
```

Comment:
| The term './scripts/loadenv.ps1' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included, verify that 
| the path is correct and try again.

Reply:
ok, as for loadenv.sh I cut the first 35 lines of prepdocs.ps1 and I put them in a new loadenv.ps1 file, then I added a new line in prepdocs.ps1
`. ./scripts/loadenv.ps1`